kvui.py

```diff
@@ -748,9 +763,12 @@ def refresh_hints(self, hints):
                 "entrance": {"text": self.parser.handle_node({"type": "color" if hint["entrance"] else "text",
                                                               "color": "blue", "text": hint["entrance"]
                                                               if hint["entrance"] else "Vanilla"})},
-                "found": {
+                "status": {
                     "text": self.parser.handle_node({"type": "color", "color": "green" if hint["found"] else "red",
```

Comment:
same comment about colors

Reply:
same reply about what color to use

Reply:
Same vague clarification that lets you make the final choice